apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -137,8 +137,14 @@ class PageTemplateModal extends Component {
 		if ( event.target.matches( 'button.template-selector-item__label' ) ) {
 			return false;
 		}
-		this.setState( { isOpen: false } );
+
 		trackDismiss( this.props.segment.id, this.props.vertical.id );
+
+		if ( typeof window.calypsoifyGutenberg !== 'undefined' ) {
+			window.top.location = window.calypsoifyGutenberg.closeUrl;
```

Comment:
We're assuming `closeUrl` is defined. We should also check it is available before we attempt to use it as otherwise the user will be sent to a url `undefined`.

Reply:
I've tried a different approach and explained in a comment